rest-api/src/main/resources/definitions/abstract_study_participant.yml

```diff
@@ -90,4 +90,7 @@ properties:
         x-nullable: false
     orgMembership:
         type: string
-        description: The identifier of the organization this account is a member of (this will be an administrative account and not a study participant). Once set, it cannot be changed on an update (there are separate organization membership APIs to change organizational membership).
\ No newline at end of file
+        description: The identifier of the organization this account is a member of (this will be an administrative account and not a study participant). Once set, it cannot be changed on an update (there are separate organization membership APIs to change organizational membership).
+    note:
+        type: string
+        description: Optional note related to user account.
```

Comment:
I think this was correct before, in that AbstractStudyParticipant is a superclass of SignUp, and SignUp shouldn't be used to add an account unless a would-be participant is signing up. In that case, they don't have the role to add a note (which should apply on create as well as update).

Reply:
As discussed, I just committed an edit to the note field description to clarify that it's only accessible from administrative accounts. Please check it out when you have a moment. I made the same change to account.yml for consistency.